src/__test__/components/HomeDetails.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ArticleDetails } from '../../components/HomePage/HomeDetail';
+
+
+const props = {
+  getArticleActions: jest.fn(),
+  match: { params: { slug: 'test' } },
+};
+
+describe('App', () => {
+  it('should match snapshot', () => {
+    const wrapper = shallow(<ArticleDetails {...props} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('should show loading when retriving an articles', () => {
```

Comment:
please, change the word `articles` to singular.

Reply:
this has been worked on @MuhanguziDavid